docs/concepts/fields.md

```diff
@@ -667,7 +667,8 @@ print(repr(Model.model_validate({'pet': {'pet_kind': 'dog', 'age': 12}})))
 #> Model(pet=Dog(pet_kind='dog', age=12))
 ```
 
-See the [Discriminated Unions] for more details.
+You can also take advantage of `Annotated` to define your discriminated unions.
+See the [Discriminated Unions] docs for more details.
```

Comment:
Does this need a link (like, `[Discriminated Unions](#discriminated_unions)` or whatever)? Or is there something magical happening with it auto-generating links from just square brackets? I recognize it was like this in current docs but seems like it's worth fixing if it's just a broken link now.

Reply:
Ah I see it can be specified at the bottom, nevermind.

Reply:
The link is at the bottom of the page :)